bridge/core/geometry/dom_matrix_read_only.cc

```diff
@@ -0,0 +1,514 @@
+
+DOMMatrixReadOnly::DOMMatrixReadOnly(webf::ExecutingContext* context, webf::NativeBindingObject* native_binding_object)
+    : BindingObject(context->ctx(), native_binding_object) {}
+
+double DOMMatrixReadOnly::getMatrixProperty(const AtomicString& prop) const {
+  NativeValue dart_result = GetBindingProperty(prop, FlushUICommandReason::kDependentsOnElement, ASSERT_NO_EXCEPTION());
+  return NativeValueConverter<NativeTypeDouble>::FromNativeValue(dart_result);
+}
+
+void DOMMatrixReadOnly::setMatrixProperty(const AtomicString& prop, double v, ExceptionState& exception_state) {
+  if (DynamicTo<DOMMatrix>(this)) {
+    SetBindingProperty(prop, NativeValueConverter<NativeTypeDouble>::ToNativeValue(v), exception_state);
+  }
+}
+
+double DOMMatrixReadOnly::m11() const {
+  return getMatrixProperty(binding_call_methods::km11);
+}
+void DOMMatrixReadOnly::setM11(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km11, v, exception_state);
+}
+double DOMMatrixReadOnly::m12() const {
+  return getMatrixProperty(binding_call_methods::km12);
+}
+void DOMMatrixReadOnly::setM12(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km12, v, exception_state);
+}
+double DOMMatrixReadOnly::m13() const {
+  return getMatrixProperty(binding_call_methods::km13);
+}
+void DOMMatrixReadOnly::setM13(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km13, v, exception_state);
+}
+double DOMMatrixReadOnly::m14() const {
+  return getMatrixProperty(binding_call_methods::km14);
+}
+void DOMMatrixReadOnly::setM14(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km14, v, exception_state);
+}
+
+double DOMMatrixReadOnly::m21() const {
+  return getMatrixProperty(binding_call_methods::km21);
+}
+void DOMMatrixReadOnly::setM21(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km21, v, exception_state);
+}
+double DOMMatrixReadOnly::m22() const {
+  return getMatrixProperty(binding_call_methods::km22);
+}
+void DOMMatrixReadOnly::setM22(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km22, v, exception_state);
+}
+double DOMMatrixReadOnly::m23() const {
+  return getMatrixProperty(binding_call_methods::km23);
+}
+void DOMMatrixReadOnly::setM23(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km23, v, exception_state);
+}
+double DOMMatrixReadOnly::m24() const {
+  return getMatrixProperty(binding_call_methods::km24);
+}
+void DOMMatrixReadOnly::setM24(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km24, v, exception_state);
+}
+
+double DOMMatrixReadOnly::m31() const {
+  return getMatrixProperty(binding_call_methods::km31);
+}
+void DOMMatrixReadOnly::setM31(double v, ExceptionState& exception_state) {
+  setMatrixProperty(binding_call_methods::km31, v, exception_state);
```

Comment:
use defined_properties::km31 will be better.

Reply:
fixed